code/modules/power/tesla/coil.dm

```diff
@@ -47,7 +47,7 @@
 	flick("coilhit", src)
 	playsound(src.loc, 'sound/magic/LightningShock.ogg', 100, 1, extrarange = 5)
 	tesla_zap(src, 5, power_produced)
-	addtimer(src, "reset_shocked", 10)
+	addtimer(src, "reset_shocked", 1)
```

Comment:
Current:
* proccall - ```addtimer()```
* Potentially waking up the sleeping timing subsystem (since with no entires, it shuts itself off)
* proccall - ```fire()``` from the subsystem
* proccall - ```call()()``` for ```reset_shocked()```

Proposed:
* ```sleep(1)```
* ```reset_shocked()```